src/components/v5/common/MemberSignatureList/types.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from '~types';
+
+export interface MemberSignatureListProps {
+  items: User[];
```

Comment:
Please pass only the needed props. I don't think that the whole user object is needed to display an avatar with user name and status